src/Plugin.php

```diff
@@ -50,7 +50,7 @@ public static function init() {
     // @todo Make lightGallery properly respect srcset & sizes in JavaScript instead
     //   of duplicating that information in HTML; see
     //   https://github.com/netzstrategen/wordpress-gallerya/pull/11#issuecomment-355664739
-    if (is_plugin_active('woocommerce/woocommerce.php')) {
+    if (static::isPluginActive('woocommerce/woocommerce.php')) {
```

Comment:
Actually, it's probably much faster to not check the existence of the plugin at all and instead simply register the filter in all cases — it will simply not be called if woocommerce is not installed.

But we can leave this as-is for now.